pkg/operator/operator.go

```diff
@@ -90,20 +90,20 @@ func CmdRun() *cobra.Command {
 // RunInstall runs a CLI command
 func RunInstall(cmd *cobra.Command, args []string) {
 	c := LoadOperatorConf(cmd)
-	util.KubeCreateSkipExisting(c.NS)
-	util.KubeCreateSkipExisting(c.SA)
-	util.KubeCreateSkipExisting(c.SAEndpoint)
-	util.KubeCreateSkipExisting(c.Role)
-	util.KubeCreateSkipExisting(c.RoleEndpoint)
-	util.KubeCreateSkipExisting(c.RoleBinding)
-	util.KubeCreateSkipExisting(c.RoleBindingEndpoint)
-	util.KubeCreateSkipExisting(c.ClusterRole)
-	util.KubeCreateSkipExisting(c.ClusterRoleBinding)
+	util.KubeCreateFailExisting(c.NS)
```

Comment:
I don't think these should change. keep it KubeCreateSkipExisting.

Reply:
I mean that for all the calls in the CLI that don't even check the return value.

Reply:
@guymguym 👍 reverting unconditional invocations.
See https://github.com/noobaa/noobaa-operator/compare/39c5a4e..3ec69ef